libsupport/include/katana/EntityTypeManager.h

```diff
@@ -214,6 +214,14 @@ class KATANA_EXPORT EntityTypeManager {
     return atomic_type_name_to_entity_type_id_.count(name) == 1;
   }
 
+  std::vector<std::string> ListAtomicTypes() const {
```

Comment:
Instead of allocating a new vector, we can return an iterator pair or `StandardRange`. The iterator can be obtained using `boost::transform_iterator` that picks the first of `key-value` pair.

Reply:
Please add a TODO and address it post MVP. Will require defining an iterator-type alias in `EntityTypeManager`.